content/en/blogs/kubesphere-3.1.0-ga-announcement.md

```diff
@@ -0,0 +1,156 @@
+
+### Multi-cloud and multi-cluster management
+
+[Multi-cloud and multi-cluster management](https://kubesphere.io/docs/multicluster-management/) is one of the most important features for KubeSphere. Most importantly, it returns great benefits for every enterprise managing heterogeneous clusters across clouds. At the same time, we want to keep pushing ourselves as we notice an increasing number of users are seeking for a lightweight approach to managing Member Clusters.
+
+![multi-cluster-architecture](/images/blogs/en/kubesphere-3.1.0-ga-announcement/multi-cluster-architecture.png)
+
+To this end, we have optimized the multi-cluster feature by removing some component dependencies (for example, Redis and OpenLDAP) in Member Clusters. The steps to import Member Clusters have also been simplified with configuration validation (e.g. `jwtSecret`) added.
+
+In addition, Tower now supports high availability to give you more choices as you create a more stable and functional multi-cluster architecture.
+
+### Observability 
+
+Observability has remained a central focus for us from day one as we work to help you understand and improve the health, performance, and availability of your resources no matter where they are deployed. In the new version, we have made our observability system more powerful and comprehensive:
+
+- **Monitoring**. You can configure `ServiceMonitor` resources on the console. PromQL auto-completion and syntax highlighting are supported now. Custom monitoring dashboards can be created at the cluster level.
+- **Alerting**. The entire alerting system has been refactored without any dependency on MySQL, Redis and etcd. In KubeSphere v3.1.0, we use Thanos Ruler and the build-in rules of Prometheus.
+- **Notification**. Notification Manager v1.0.0 is integrated into KubeSphere. Different notification methods can be configured on the KubeSphere console directly, including email, DingTalk, Slack, WeCom and Webhook.
+- **Logging**. Support kubelet, Docker, and containerd log collection. Logs can be exported to [Loki](https://github.com/kubesphere/fluentbit-operator/blob/master/docs/plugins/output/loki.md).
+
+![alerting](/images/blogs/en/kubesphere-3.1.0-ga-announcement/alerting.png)
+
+![notification](/images/blogs/en/kubesphere-3.1.0-ga-announcement/notification.png)
+
+### DevOps
+
+[KubeSphere DevOps](https://kubesphere.io/docs/devops-user-guide/understand-and-manage-devops-projects/overview/) is one of the most used - if not the most used - functions among KubeSphere users as it automates software release processes. Since its integration into KubeSphere, it has greatly increased enterprises' ability to deliver applications and services with speed and security.
+
+The new KubeSphere DevOps System has been further stabilized to improve user experiences. In the new version, you can:
+
+- Clone a pipeline.
+- Run pipelines in a batch.
+- Create a multi-branch pipeline with GitLab.
```

Comment:
I assume this link can also be added: https://kubesphere.io/docs/devops-user-guide/how-to-use/gitlab-multibranch-pipeline/